frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_list_item.tsx

```diff
@@ -339,6 +371,58 @@ function _SegmentListItem({
         andCloseContextMenu,
       )}
       {getMakeSegmentActiveMenuItem(segment, setActiveCell, activeCellId, andCloseContextMenu)}
+      <Menu.Item key="changeSegmentColor" disabled={isEditingDisabled || segment.id !== mappedId}>
```

Comment:
Good point, I just added this comment:

```
/*
       * Disable the change-color menu if the segment was mapped to another segment, because
       * changing the color wouldn't do anything as long as the mapping is still active.
       * This is because the id (A) is mapped to another one (B). So, the user would need
       * to change the color of B to see the effect for A.
       */
```